dockerfile/rocm5.1.x.dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM rocm/pytorch:rocm5.1.1_ubuntu20.04_py3.7_pytorch_1.10.0
+ARG BASE_IMAGE=rocm/pytorch:rocm5.1.3_ubuntu20.04_py3.7_pytorch_1.11.0
+FROM ${BASE_IMAGE}
+
+# 5.1.x base images:
+# rocm5.1.1 - rocm/pytorch:rocm5.1.1_ubuntu20.04_py3.7_pytorch_1.10.0
+# rocm5.1.3 - rocm/pytorch:rocm5.1.3_ubuntu20.04_py3.7_pytorch_1.11.0
 
 # OS:
 #   - Ubuntu: 20.04
 #   - OpenMPI: 4.0.5
 #   - Docker Client: 20.10.8
 # ROCm:
-#   - ROCm: 5.1.1
-#   - RCCL: 2.11.4
-#   - RCCL RDMA SHARP plugins: rocm-rel-5.1
-#   - hipify: 5.1.1
+#   - ROCm: 5.1.x
+#   - RCCL: 2.12.10+6707a27
+#   - hipify: 5.1.x
 # Mellanox:
 #   - OFED: 5.2-2.2.3.0
```

Comment:
do we need to update ofed to 5.4 or 5.6 for Ubuntu 20.04?

Reply:
not sure about that, but the ofed versions and os versions should be orthogonal? there're ofed {5.2, 5.4, 5.6} x ubuntu {18.04, 20.04} combinations according to https://linux.mellanox.com/public/repo/mlnx_ofed/5.2-2.2.3.0/ubuntu20.04/x86_64/